src/fsharp/FSharp.Build-proto/FSharp.Build-proto.fsproj

```diff
@@ -1,57 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm uncomfortable with removing the "proto" build project files if it means that http://github.com/Microsoft/visualfsharp is  _forever_ and _completely_ going down a path that requires a CoreCLR-based proto compiler, and forever cutting the repo off from a bootstrap process that involves .NET 4.x only.  

For example, a Mono build will require these proto build projects (they are used in http://github.com/fsharp/fsharp).  So if we remove these files, it makes it much harder to integrate the remaining changes back from http://github.com/fsharp/fsharp and get a Mono build+test established in this repo.  My feeling is that F#+Mono is not going away any time soon, regardless of what we believe about CoreCLR.

There have also been people complaining about the forced CoreCLR dependency for the build process on Windows - e.g. people on slow internet connections or behind firewalls.

So I'd recommend keeping these files, and look to establish .NET Framework and Mono builds in this repo.  Or at least leave the question open.